README.md

```diff
@@ -18,10 +18,11 @@ Docker images can be found in Expedia Group's [dockerhub](https://hub.docker.com
 
 Beekeeper makes use of [Apiary](https://github.com/ExpediaGroup/apiary) - an open source federated cloud data lake - to detect changes in the Hive Metastore. One of Apiary’s components, the [Apiary Metastore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/main/apiary-metastore-events/sns-metastore-events/apiary-metastore-listener), captures Hive events and publishes these as messages to an SNS topic. Beekeeper uses these messages to detect changes to the Hive Metastore, and perform appropriate deletions.
 
-Beekeeper is comprised of three separate Spring-based Java applications:
+Beekeeper is comprised of four separate Spring-based Java applications:
```

Comment:
Might be helpful to include a contents table with hyperlinks at the top of this README with all the titles of the sections, that way you can quickly see whats included and jump straight there